client/src/components/dashboard/Sidebar.tsx

```
@@ -0,0 +1,75 @@
+  {!isOpen && (
+    <div className="absolute -right-4 top-1/2 -translate-y-1/2 w-28 h-16 bg-purple-900 rounded-r-full rounded-l-[40px] shadow-lg"></div>
+  )}
+
+  {/* Sidebar Toggle Button */}
+  {!isOpen && (
+    <button
+      onClick={toggleSidebar}
+      className="relative p-3 bg-pink-600 text-white rounded-full shadow-lg z-10"
+      aria-label="Open Sidebar"
+      title="Open Sidebar"
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)